pilot/pkg/model/context.go

```diff
@@ -517,6 +517,10 @@ const (
 
 	// NodeMetadataInstanceIPs is the set of IPs attached to this proxy
 	NodeMetadataInstanceIPs = "INSTANCE_IPS"
+
+	// NodeMetadataSdsTokenPath specifies the path of the SDS token used by the Enovy proxy.
+	// If not set, Pilot uses the default SDS token path.
```

Comment:
That doesn't seem to match the implementation - with current code is true only if some install options are set.

But I think the comment is right - and the implementation needs to be fixed to match it, i.e. if set Envoy will use 
the specified SDS token path ( independent of the global defaults )

Reply:
Done: implementation has been revised to match it.